portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/FileInstall.testcase

```diff
@@ -115,6 +117,8 @@ definition {
 
 		GogoShell.openGogoShellAdmin();
 
+		Pause(locator1 = "10000");
```

Comment:
Are these extra pauses still needed?

Reply:
yes, the pause is needed otherwise the test will keep failing.